src/v3/src/util/languageUtils.ts

```diff
@@ -11,6 +11,9 @@
  */
 
 import { OdysseyI18nResourceKeys, odysseyI18nResourceKeysList } from '@okta/odyssey-react-mui';
+// i18next is already included in Odyssey
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
We should add it to our own dependencies. The eslint warning/error you're suppressing here is saying that we're directly depending on a transitive, i.e., Odyssey could remove/replace i18next and it'd break our app.